akka-sample-cluster-client-grpc-java/pom.xml

```diff
@@ -18,37 +18,37 @@
   <dependencies>
     <dependency>
       <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-stream_2.12</artifactId>
+      <artifactId>akka-stream_2.13</artifactId>
```

Comment:
I think it's a good moment to extract the scala binary version to a pom `<property>`.

Reply:
It doesn't really change often, and for Java users it's more of a "use this artifact" than think of it as a Scala version.

Reply:
Fair enough. I had https://github.com/lightbend/paradox/pull/429 in mind. LGTM